lib/capistrano/tasks/python.cap

```diff
@@ -0,0 +1,12 @@
+set :release_version, ask('Enter the release tag:', 'master')
```

Comment:
I think the prompt is useful, I'm just getting a weird result because of the `ask`:
```
...
ruby 2.3.4p301 (2017-03-30 revision 58214) [x86_64-linux]
Please enter Enter the release tag: (master):
00:00 python:update_cli
...
```